src/nile/core/account.py

```diff
@@ -16,6 +16,7 @@
 from nile.core.call_or_invoke import call_or_invoke
 from nile.core.declare import declare
 from nile.core.deploy import deploy
+from nile.core.deploy import deploy_contract as deploy_with_udc
```

Comment:
```suggestion
from nile.core.deploy import deploy_contract as deploy_with_deployer
```
for consistency with the above suggestion